community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/PlannerName.scala

```diff
@@ -60,6 +60,13 @@ case object DPPlannerName extends CostBasedPlannerName {
   val name = "DP"
 }
 
+/**
+  * Procedures are not really planned, b
+  */
+case object ProcedurePlannerName extends CostBasedPlannerName {
```

Comment:
This should not be a CostBasedPlannerName. Perhaps inherit directly from PlannerName?